backends-velox/src/test/scala/io/glutenproject/execution/VeloxParquetWriteSuite.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.glutenproject.execution
+
+class VeloxParquetWriteSuite extends WholeStageTransformerSuite {
+  override protected val backend: String = "velox"
+  override protected val resourcePath: String = "/tpch-data-parquet-velox"
+  override protected val fileFormat: String = "parquet"
+
+  override def beforeAll(): Unit = {
+    super.beforeAll()
+    createTPCHNotNullTables()
+  }
+
+  test("test write parquet with compression codec") {
+    // compression codec details see `VeloxParquetDatasource.cc`
+    Seq("snappy", "gzip", "zstd", "none", "uncompressed")
```

Comment:
velox seems does not support `lz4` at read side

Reply:
Does it throw unsupport exception with lz4? Velox parquet writer call arrow writer to write the RowVector into buffer? Maybe arrow does not support lz4. I will check it later.

Reply:
yes, it seems due to velox page reader, see `velox/dwio/parquet/reader/PageReader.cpp`.